recipes/glue-vispy-viewers/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.3" %}
+
+package:
+  name: glue-vispy-viewers
+  version: {{version}}
+
+source:
+  fn: glue-vispy-viewers-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/g/glue-vispy-viewers/glue-vispy-viewers-{{version}}.tar.gz
+  md5: 507d396069cb2eb96fc2e0c3058f88c3
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py35]
+
+requirements:
+
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - vispy
+    - glueviz
+    - scikit-image
+    - matplotlib
+
+test:
+  requires:
+    - mock
+    - pytest
+  imports:
+    - glue_vispy_viewers
+
+about:
+  home: https://github.com/glue-viz/glue-vispy-viewers
+  license: BSD 3-Clause
+  summary: 3D viewers for Glue
+
+extra:
+  recipe-maintainers:
+    - astrofrog
```

Comment:
@astrofrog this recipe seems to be good to go. I just recommend you to invite more maintainers to share the load. (Can be done later in the feedstock.)